pkg/reader/ccip.go

```diff
@@ -98,6 +102,16 @@ func (r *ccipChainReader) WithExtendedContractReader(
 	return r
 }
 
+func (r *ccipChainReader) Close() error {
+	// Stop the background poller
+	r.configPoller.Stop()
+	r.lggr.Info("Stopped config background polling")
```

Comment:
nit: log belongs better probably in `configPoller.Stop()` method

Reply:
another nit: to be consistent maybe `Stop` can be called `Close` instead? Also makes it compatible with `io.Closer`